applications/nrf5340_audio/src/audio/sw_codec_select.h

```diff
@@ -44,6 +44,7 @@ enum sw_codec_channel_mode {
 
 struct sw_codec_encoder {
 	bool enabled;
+	bool started;
```

Comment:
I may see later why this started is required. 
It is often better with a state machine instead of two variables.